README.md

```diff
@@ -17,7 +50,7 @@ or if you want to use the `marked` CLI tool (not necessary when using npm run-sc
 npm install -g marked
 ```
 
-## Usage
+<h2 id="usage">Usage</h2>
```

Comment:
I think github auto generates IDs on headings? It should be safe to reference #usage inside the document

Reply:
Could be fair. Was mentioned in the review on #1081 as well. I've always run into issues there; so, normally end up doing it by hand. Also, could make it more difficult to use in a non-GFM setting; at least here we know the TOC will always work.